src/GRA.Controllers/MissionControl/RolesController.cs

```diff
@@ -20,47 +20,53 @@ public class RolesController : Base.MCController
         private readonly ILogger<RolesController> _logger;
         private readonly AuthorizationCodeService _authorizationCodeService;
         private readonly RoleService _roleService;
+        private readonly UserService _userService;
+        public static string Name { get { return "Roles"; } }
+
         public RolesController(ILogger<RolesController> logger,
             ServiceFacade.Controller context,
             AuthorizationCodeService authorizationCodeService,
-            RoleService roleService)
+            RoleService roleService,
+            UserService userService)
             : base(context)
         {
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
             _authorizationCodeService = authorizationCodeService
                 ?? throw new ArgumentNullException(nameof(authorizationCodeService));
             _roleService = roleService ?? throw new ArgumentNullException(nameof(roleService));
+            _userService = userService ?? throw new ArgumentNullException(nameof(userService));
         }
 
         #region Roles
-        public async Task<IActionResult> Index(int page = 1)
+
+        public async Task<IActionResult> Index(int page)
         {
-            var filter = new BaseFilter(page);
+            var filter = new BaseFilter(page == default ? 1 : page);
 
             var roleList = await _roleService.GetPaginatedListAsync(filter);
 
-            PaginateViewModel paginateModel = new PaginateViewModel
+            var paginateModel = new PaginateViewModel
             {
                 ItemCount = roleList.Count,
-                CurrentPage = page,
+                CurrentPage = page == default ? 1 : page,
                 ItemsPerPage = filter.Take.Value
             };
+
             if (paginateModel.PastMaxPage)
             {
-                return RedirectToRoute(
-                    new
-                    {
-                        page = paginateModel.LastPage ?? 1
-                    });
+                return RedirectToRoute(new { page = paginateModel.LastPage ?? 1 });
             }
 
-            var viewModel = new RoleListViewModel
+            var viewmodel = new RoleListViewModel
```

Comment:
Why was the camelcase removed on this?

Reply:
I got lazy and tired of hitting the shift key ;)